### ISSUE TITLE
Ability to control module-order in appups

### ISSUE THREAD

**Issue by RJ** [RJ/relflow#2]
Not possible at the moment. Have to manually edit appups after generation.

Could use a module attribute like:

```
-relflow_priority(100).
```

or perhaps:

```
-relflow_depends(other_util_module).
```

Check how knit does this (per comment in README).

**Comment by ferd** [comment]
I'm guessing a general way to do it would be to use xref to create a directed graph of dependencies, and order the operations using that. 

Modules that nothing depend on, but depend on many others get updated first, given they likely have code to handle the transition period without crashing, and you can go down the graph until all is covered.

 It may still require for someone to go through the files to make sure they're okay, but might do the right team the majority of the time without intervention. This leaves a hole where people still have to go and figure out when to do suspend/resume cycles anyway, and I'm not sure those are easy to sort out.

**Comment by RJ** [comment]
good point! xref analysis would probably give a decent solution in most cases :+1:
